src/index.js

```diff
@@ -1,2 +1,57 @@
+/* eslint-disable no-console */
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+
+  return element;
+}
+const initialCount = 0;
+
+function render(count) {
+  function handleClick() {
+    render(count + 1);
+  }
+
+  function handleClickNumber(value) {
+    if (Number.isNaN(Number(value))) {
+      const errorMsg = 'Not Numbers';
+      return console.assert(false, { Number: value, errorMsg });
```

Comment:
return 하는 것을 보면 `assert`의 결과를 반환하는 것처럼 보여요. 명확하게 작성해야 합니다.
그리고 Msg라기보다 Message라고 명확하게 써주는게 더 좋을 것 같아요

```suggestion
      console.assert(false, { Number: value, errorMsg });
      return 
```

Reply:
msg: 글루탐산 나트륨

https://ko.m.wikipedia.org/wiki/%EA%B8%80%EB%A3%A8%ED%83%90%EC%82%B0_%EB%82%98%ED%8A%B8%EB%A5%A8